CONTRIBUTING.md

```diff
@@ -0,0 +1,149 @@
+what you intend to implement.
+
+Once the feature is done it is merged back into `master` by making a Pull
+Request.
+
+
+### General stuff about git and commit messages
+
+In general it is better to commit often. Small commits are easier to roll back
+and also makes the code easier to review.
+
+Write helpful commit messages that describes the changes and possibly why they
+were necessary.
+
+Each commit should contain changes that are functionally connected and/or
+related. If you for example want to write _and_ in a commit message this is a
+good sign that it should have been two commits.
+
+Learn how to select chunkgs of changed files to do multiple separate commits of
+unrelated things. This can be done with either `git add -p` or `git commit -p`.
+
+
+#### Helpfull commit messages
+
+The commit messages may be seen as meta-comments on the code that are
+incredibly helpful for anyone who wants to know how this piece of software is
+working, including colleagues (current and future) and external users.
+
+Some tips about writing helpful commit messages:
+
+#. Separate subject (the first line of the message) from body with a blank line.
+#. Limit the subject line to 50 characters.
+#. Capitalize the subject line.
+#. Do not end the subject line with a period.
+#. Use the imperative mood in the subject line.
+#. Wrap the body at 72 characters.
+#. Use the body to explain what and why vs. how.
+
+For an in-depth explanation of the above points, please see [How to Write a Git
+Commit Message](http://chris.beams.io/posts/git-commit/).
+
+
+### How we do code reviews
+
+A code review is initiated by making a Pull Request in the appropriate repo on
+github.
+
+Work should not continue on the branch _unless_ it has been marked with the `DO
+NOT MERGE` tag. Once the PR is ready and the `DO NOT MERGE` tag is removed the
+review process can go on to the next stage.
+
+Before review starts it is a good idea to rebase your branch to `master` to
+ensure that we can make a clean merge.
+
+The initiator of the PR should recruit 2 reviewers that get assigned reviewer
+duty on the branch. It's not a bad idea to announce this on the #localega slack
+channel.
+
+Other people may also look at and review the code.
+
+A reviewers job is to:
+
+  * Read the code and make sure it is understandable
+  * Make sure that commit messages and commits are structured so that it is
+    possible to understand why certain changes were made.
+  * Ensure that the test-suite covers the new behavior
+
+It is _not_ the reviewers job to checkout and run the code - that is what the
+test-suite is for.
+
+Once all the reviews are positive the Pull Request can be _merged_ into
+`master` and the feature branch deleted.
+
+
+----
+
+Thanks again,
+/NBIS System Developers
+
+[searching under Issues]: https://github.com/NBISweden/LocalEGA/issues?utf8=%E2%9C%93&q=is%3Aissue%20label%3Abug%20%5BBUG%5D%20in%3Atitle
```

Comment:
```suggestion
[searching under Issues]: https://github.com/EGA-archive/LocalEGA/issues?utf8=%E2%9C%93&q=is%3Aissue%20label%3Abug%20%5BBUG%5D%20in%3Atitle
```

Reply:
I would also change the "NBIS System Developers" to "EGA System Developers" too.

Reply:
Should we do something for that repo under NBIS organisation, e.g. archive it?